### ISSUE TITLE
[FEATURE] - EXIF data

### ISSUE THREAD

**Issue by shaun-ba** [tfree87/pyRenamer#18]
The previous pyrenamer supported EXIF data, so for example when renaming images {year} would use the year the photo was taken.

Have you removed this functionality?

**Comment by TheTimeTombs** [comment]
Yes, pyRenamer used to support EXIF data. To be perfectly honest, it has been so long since I made this fork that I forgot why this feature was removed. It should be fairly simple to add that functionality back into pyRenamer. I will look into it and see if I can add that back in.

**Comment by xoir** [comment]
My understanding of this is superficial, but as far as I can see it is basically two functions from [pyrenamer_filefunctions.py](https://github.com/tfree87/pyRenamer/files/10197775/pyrenamer-0.6.0.zip)

line 413 | def replace_images(name, path, newname, newpath):
line 468 | def get_exif_data(path):

To get these working you would need EXIF.py (see the zip file) and it needs to be integrated in the GTK Frontend as well.

If I can help in any way, please let me know. I would love to see pyRenamer supporting EXIF again!
